user_agent.go

```diff
@@ -153,7 +153,7 @@ func (p *UserAgent) Parse(ua string) {
 }
 
 // Returns the mozilla version (it's how the User Agent string begins:
-// "Mozilla/5.0 ...", unless we're dealing with Opera, of course).
+// "Mozilla/5.0 ...", unless we're dealing with Opera or Dalvik, of course).
```

Comment:
That's correct. For Dalvik, your patch sets the Mozilla attribute to 5.0, as I said. However, this comment should not be modified, because for Dalvik we are also setting the proper Mozilla value. In this comment we are explicitly saying that only Opera has an undefined Mozilla version.